res/values/strings.xml

```diff
@@ -1677,46 +1664,46 @@
+    <string name="backup_enable_dialog__i_have_written_down_this_passphrase">I have written down this passphrase. Without it, I will be unable to restore from a backup.</string>
+    <string name="registration_activity__restore_backup">Restore from backup</string>
     <string name="registration_activity__skip">Skip</string>
     <string name="registration_activity__register">Register</string>
-    <string name="preferences_chats__chat_backups">Chat backups</string>
-    <string name="preferences_chats__backup_chats_to_external_storage">Backup chats to external storage</string>
-    <string name="preferences_chats__create_backup">Create backup</string>
+    <string name="preferences_chats__chat_backups">Let Signal create backups</string>
+    <string name="preferences_chats__backup_chats_to_external_storage">Allow Signal to create encrypted backup files of your chats and media to /Signal/Backups on your phone\'s internal storage. If enabled Signal will create backups automatically every 24 hours. Only the two most recent successful backups will be kept.</string>
+    <string name="preferences_chats__create_backup">Create backup file now</string>
     <string name="RegistrationActivity_enter_backup_passphrase">Enter backup passphrase</string>
     <string name="RegistrationActivity_restore">Restore</string>
-    <string name="RegistrationActivity_backup_failure_downgrade">Cannot import backups from newer versions of Signal</string>
+    <string name="RegistrationActivity_backup_failure_downgrade">Cannot import backups which were created by a newer version of Signal</string>
     <string name="RegistrationActivity_incorrect_backup_passphrase">Incorrect backup passphrase</string>
     <string name="RegistrationActivity_checking">Checking…</string>
     <string name="RegistrationActivity_d_messages_so_far">%d messages so far…</string>
-    <string name="RegistrationActivity_restore_from_backup">Restore from backup?</string>
-    <string name="RegistrationActivity_restore_your_messages_and_media_from_a_local_backup">Restore your messages and media from a local backup. If you don\'t restore now, you won\'t be able to restore later.</string>
+    <string name="RegistrationActivity_restore_from_backup">Restore from backup file?</string>
+    <string name="RegistrationActivity_restore_your_messages_and_media_from_a_local_backup">You now have the opportunity to restore messages and media from a backup file. Beware: Restoring from a backup file can only be done during the installation process of Signal. You can not restore from a backup file once you\'ve started using Signal, so if you wish to restore from a backup file you must do so now.</string>
```

Comment:
I don't like "installation process of Signal". It would be the setup or the registration process. Anyway, it could be better ways to ephaise this without repetition. For example "Restore your messages and media from a backup file. This is the only opportunity to restore the data. If you don\'t restore now, you won\'t be able to restore later."

Reply:
well you can do it later, if you reinstall.

Reply:
I don't think that subtlety is important here vs alerting the user of the importance of restoring the backup at that moment. I don't feel comfortable guarantee that you can still restore the backup reinstalling the app, but after passing this screen. For example, the backup can be enabled again and the old files could be overwritten.